test/unit/tableDesigner.test.ts

```diff
@@ -445,42 +445,6 @@ suite("TableDesignerWebviewController tests", () => {
         );
     });
 
-    test("should call initialize in initializeTableDesigner reducer", async () => {

```

Comment:
Removing these tests as they test only reducer functionality and testing this now doesn't make much sense as we have switched to JSON RPC library.